src/RenderingSystem.cpp

```diff
@@ -86,32 +86,32 @@ bool RenderingSystem::init_window() {
     const int screen_width = 640;
     const int screen_height = 480;
 
+    SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
+    SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
+    SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
     window_ = SDL_CreateWindow("WienerTakesAll",
-                               SDL_WINDOWPOS_UNDEFINED,
-                               SDL_WINDOWPOS_UNDEFINED,
-                               screen_width,
-                               screen_height,
-                               sdl_flags);
+                              SDL_WINDOWPOS_UNDEFINED,
+                              SDL_WINDOWPOS_UNDEFINED,
+                              screen_width,
+                              screen_height,
+                              sdl_flags);
 
     if (window_ == NULL) {
         std::cout << "Window could not be created! SDL Error: " << SDL_GetError() << std::endl;
         return false;
     }
 
 
-    SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
-    SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-    SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
```

Comment:
Why was this file changed?

Reply:
On my virtual machine GLEW wasn't getting the right OpenGL versions, failing with "Missing GL version" and eventually crashing with a segfault. From what I gathered, certain SDL_GL_* calls need to be called before or after the context is created to prevent this from happening.